app/backend/src/clj/teet/file/file_db.clj

```diff
@@ -451,6 +453,27 @@
           :in $ ?uuid]
         db uuid)))
 
+;; used by vektorio scheduled upload retry. considering only files that are older than threshold,
+;; so we don't step on top of normally (by s3 trigger) started uploads.
+(defn recent-task-files-without-model-id [db threshold-in-minutes]
```

Comment:
Should this also check that the files belong to a not-deleted task/activity?

Reply:
Yep, I guess this PR is a fine place to handle this as well. I added these checks to clojure code using a separate file-db/is-task-file? fn now to both normal and scheduled import code paths.